sql/mysql/Positive-Technologies/MySqlLexer.g4

```diff
@@ -245,6 +253,7 @@ VARBINARY:                           'VARBINARY';
 TINYBLOB:                            'TINYBLOB';
 BLOB:                                'BLOB';
 MEDIUMBLOB:                          'MEDIUMBLOB';
+LONG:                                'LONG';
```

Comment:
@KvanTTT this LONG is used for LONG VARCHAR and LONG VARBINARY. maybe it's better to give the token a better fitting name and split between the 2 usages?

Reply:
I don't exactly understand. What are you suggesting? To introduce tokens `LONG VARCHAR` and `LONG VARBINARY` instead of single `LONG`?

Reply:
Exactly. From a database view VARCHAR is linked to encoding and VARBINARY is arbitrary bytes. I was suggesting that the ANTLR grammar should maybe separate this as it might have semantical differences, though technically (like transport level) might be the same.

Reply:
I am not sure about making it a combined a combine token but I agree with the `VARBINARY` problem. I'll prepare a separate PR to handle it differently.

Reply:
@rk3rn3r  Please see followup - https://github.com/antlr/grammars-v4/pull/1796